app/models/user.rb

```diff
@@ -1,25 +1,25 @@
 class User < ApplicationRecord
-    has_secure_password
+  self.table_name = 'myusers'
 
-    self.table_name = 'myusers'
-  
-    validates :name, presence: true
-    validates :post_counter, numericality: { only_integer: true, greater_than_or_equal_to: 0 }
-  
-    has_secure_password
-  
-    has_many :posts, foreign_key: :author_id
-    has_many :comments
-    has_many :likes
-  
-    attribute :name, :string
-    attribute :bio, :text
-    attribute :post_counter, :integer, default: 0
-    attribute :photo, :string
-  
-    def recent_posts
-      posts.order(created_at: :desc).limit(3)
-    end
-  
+  has_secure_password
+
+  validates :name, presence: true
+
+  validates :name, presence: true
+  validates :post_counter, numericality: { only_integer: true, greater_than_or_equal_to: 0 }
+
+  has_secure_password
+
+  has_many :posts, foreign_key: :author_id
+  has_many :comments
+  has_many :likes
+
+  attribute :name, :string
+  attribute :bio, :text
+  attribute :post_counter, :integer, default: 0
+  attribute :photo, :string
+
+  def recent_posts
+    posts.order(created_at: :desc).limit(3)
   end
```

Comment:
- [ ] Great job so far @SAZZAD1Q2   There are a few changes I will suggest you implement and you'll be on your way to the next Milestone :wink: 
- [ ] Great job adding most of the tests, however, I noticed that per the project requirements, you were asked to `Add unit specs for all of your models' methods and validations.` But currently, this `recent_posts` method in your `User Model` hasn't been tested. So kindly include tests for that method as well :wink: